app/index.html

```diff
@@ -82,6 +82,7 @@
     <script src="bower_components/angular-bootstrap/ui-bootstrap-tpls.js"></script>
     <script src="bower_components/lodash/lodash.js"></script>
     <script src="bower_components/bootstrap/dist/js/bootstrap.js"></script>
+    <script src="bower_components/bootstrap-combobox/js/bootstrap-combobox.js"></script>
```

Comment:
Curious if they have considered the idea of "patternfly core" and "patternfly extensions"... just installing the core but giving a list of optional addons that don't land in your bower_components by default.